ouroboros-network/src/Ouroboros/Network/PeerSelection/LedgerPeers.hs

```diff
@@ -101,12 +101,6 @@ instance Show TraceLedgerPeers where
     show FallingBackToBootstrapPeers = "Falling back to bootstrap peers"
     show DisabledLedgerPeers = "LedgerPeers is disabled"
 
--- | A relay can have either an IP address and a port number or
--- a domain with a port number
-data RelayAddress = RelayDomain !DomainAddress
-                  | RelayAddress !IP.IP !Socket.PortNumber
-                  deriving (Show, Eq, Ord)
-
```

Comment:
Had to remove it from `LedgerPeers` and define it in `RootPeersDNS` because of circular dependencies